### ISSUE TITLE
HOW TO SET STOP_MARKET ORDER 

### ISSUE THREAD

**Issue by ali203** [sammchardy/python-kucoin#94]
witch parameter we use to set stop_market order : create_limit_order(symbol, side, price, size, client_oid=None, remark=None, time_in_force=None, stop=None, stop_price=None, stp=None, cancel_after=None, post_only=None, hidden=None, iceberg=None, visible_size=None)

there's no parameter named type 

thank you :)

**Comment by SyntaxJO** [comment]
Yes, I am also having trouble with this one. The only thing I can do is place a limit order, but this ties up my funds and other stop possibilities. Would love a solution to this.

**Comment by ali203** [comment]
sorry bro I still don't know , maybe you can make it manually(code) like me
if condition True then sell market
good luck 😃
